kubemarine/procedures/do.py

```diff
@@ -60,7 +66,7 @@ def main(cli_arguments=None):
     remote_args = []
 
     if '--' not in cli_arguments:
-        remote_args = cli_arguments
+        kubemarine_args = cli_arguments
```

Comment:
Previously do whoami was possible, but now only do -- whoami is possible. It is potentially possible to persist previous API, but we need to decide whether it is necessary.

Reply:
We've describe this behaviour nowhere, for this reason we can change this logic.
Without it we can't throw help section with command `do -h`, because `-h` will be parsed as part of remote command.